backend/app/views/spree/admin/search/users.json.jbuilder

```diff
@@ -4,7 +4,13 @@ json.array!(@users) do |user|
   json.id user.id
   json.email user.email
 
-  address_fields = [:firstname, :lastname, :address1, :address2, :city, :zipcode, :phone, :state_name, :state_id, :country_id, :company]
+  address_fields = [
+    :name, :address1, :address2, :city, :zipcode, :phone, :state_name,
+    :state_id, :country_id, :company
+  ]
```

Comment:
I'd take advantage of this change to move one attribute per line here, so we won't change git history for all of them if only one changes in the future.